src/featurelist.cpp

```diff
@@ -45,7 +45,7 @@ bool FeatureList::protectSelectedAppsSupported() const {
 }
 
 bool FeatureList::captivePortalNotificationSupported() const {
-#if !defined(MVPN_LINUX) || defined(MVPN_MACOS) || defined(MVPN_WINDOWS) || \
+#if defined(MVPN_LINUX) || defined(MVPN_MACOS) || defined(MVPN_WINDOWS) || \
```

Comment:
I was not quite shure about this one? - We're supporting CP on linux right? 

This evaluates to true on android, since mvpn_linux is not defined.